### PR TITLE
 Add ProximityPlacementGroup to azure_rm_availabilityset

### PR DESCRIPTION
**Pull request by nbr23**
##### SUMMARY

 Add ProximityPlacementGroup to azure_rm_availabilityset

##### ISSUE TYPE

- Module enhancement PR

##### COMPONENT NAME

azure_rm_availabilityset

### CODE REVIEW

**Comment by Fred-sun** [comment]
@nbr23 Thanks for your contribution. Can you add test cases for the new parameters? Thank you very much!

File: plugins/modules/azure_rm_availabilityset.py [hunk 1]
```diff
@@ -235,10 +254,18 @@ def exec_module(self, **kwargs):
...
+                proximity_placement_group = self.get_proximity_placement_group(parsed_proximity_placement_group.get('resource_group', self.resource_group),
+                                                                               parsed_proximity_placement_group.get('name'))
+                self.proximity_placement_group_resource = self.compute_models.SubResource(id=proximity_placement_group.id)
+                proximity_placement_group_id = proximity_placement_group.id.lower()
+
             if not response:
                 to_be_updated = True
             else:
                 update_tags, response['tags'] = self.update_tags(response['tags'])
+                response_proximity_placement_group = response['proximity_placement_group'].lower() if response.get('proximity_placement_group') is not None else None
```
Comment (Fred-sun) [inline plugins/modules/azure_rm_availabilityset.py]: line too long (165 > 160 characters)

File: plugins/modules/azure_rm_availabilityset.py [hunk 2]
```diff
@@ -331,11 +362,17 @@ def get_availabilityset(self):
...
+
 
 def main():
     """Main execution"""
     AzureRMAvailabilitySet()
 
 
 if __name__ == '__main__':
-    main()
+    main()
```
Comment (Fred-sun) [inline plugins/modules/azure_rm_availabilityset.py]: no newline at end of file

**Comment by nbr23** [comment]
Sure thing, working on it, thanks!

**Comment by nbr23** [comment]
Testing updated and validated locally. Let me know if we need more.
Thanks!

**Pull request merged**